components/tabcordion/src/Tabcordion.js

```diff
@@ -87,13 +98,13 @@ export const Tabcordion = props => {
 	const tabCount = Children.count(props.children);
 
 	return (
-		<div ref={containerRef}>
+		<div ref={containerRef} css={{ ...overrides.CSS }}>
```

Comment:
Nice but we said we would lowercase `css` when it stands alone: https://github.com/WestpacGEL/GEL/tree/develop#local-tokens-naming-convention-overwrites